Cargo.toml

```
@@ -20,7 +20,7 @@ rustdoc-args = ["--cfg", "docsrs"]
 [dependencies]
 lazy_static = "1"
 lock_api = { version = "0.4", optional = true }
-parking_lot = { version = "0.11", optional = true }
+parking_lot = { version = "0.12", optional = true }
```

I read the changelog, but there doesn't appear to be anything breaking API wise, no? So I'd prefer to change it like this:

```suggestion
parking_lot = { version = ">= 0.11, < 0.13", optional = true }
```

That way we don't have to introduce a breaking change just yet.

---

Never mind, this will always pull in 0.12 as long as the dependencies can be met so that doesn't work. Ignore this suggestion.